CHANGELOG.md

```diff
@@ -26,8 +26,7 @@
 #### New Features
 
 - Added limited support for the `Timedelta` type, including
-  - support `copy`, `cache_result`, `shift`, `sort_index`.
-  - support `astype` from `int` to `Timedelta`
+  - support `astype`, `copy`, `cache_result`, `shift`, `sort_index`.
```

Comment:
I prefer the older version with `support `astype` from `int` to `Timedelta``. We support tracking the timedelta type through `cache_result`, `shift`, etc., while `astype(pd.Timedelta)` is about converting non-timedeltas to timedelta.

Reply:
Done.